content/en/blog/_posts/2023-08-04-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: blog
+title: "Kubernetes v1.28: Planternetes"
+date: 2023-08-15T12:00:00+0000
+slug: kubernetes-v1-28-release
+---
+
+**Authors**: [Kubernetes v1.28 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.28/release-team.md)
+
+Announcing the release of Kubernetes v1.28 Planternetes, the second release of 2023!
+
+This release consists of 46 enhancements. 20 of those enhancements are entering Alpha, 14 have graduated to Beta, and 12 have graduated to Stable.
```

Comment:
We got 45 Enhancements, right? 

The KEP https://github.com/kubernetes/enhancements/issues/4138 is a repost from https://github.com/kubernetes/enhancements/issues/3085 and both appear on the project board https://github.com/orgs/kubernetes/projects/140/views/1

Reply:
Thanks, done.